Neos.Flow/Classes/Core/Booting/Scripts.php

```diff
@@ -477,12 +480,12 @@ public static function initializeSystemFileMonitor(Bootstrap $bootstrap)
         ];
 
         $context = $bootstrap->getContext();
-        /** @var PackageManagerInterface $packageManager */
+        /** @var PackageManager $packageManager */
         $packageManager = $bootstrap->getEarlyInstance(PackageManagerInterface::class);
         $packagesWithConfiguredObjects = static::getListOfPackagesWithConfiguredObjects($bootstrap);
 
         /** @var PackageInterface $package */
-        foreach ($packageManager->getAvailablePackages() as $packageKey => $package) {
+        foreach ($packageManager->getFlowPackages() as $packageKey => $package) {
```

Comment:
In the following lines we refer to `getConfigurationPath()`, `getResourcesPath()` and `getAutoloadPaths()` but they are not part of the `PackageInterface` any longer. The latter isn't even part of the `FlowPackageInterface`

Reply:
True, the docblock is bad.